app/views/ProfileView.scala.html

```diff
@@ -0,0 +1,40 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    layout: templates.Layout,
+    govukSummaryList: GovukSummaryList
+)
+
+@(detailsList: SummaryList)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = titleNoForm(messages("profile.title"))) {
+
+    <h1 class="govuk-heading-l">@messages("profile.h1")</h1>
+
+    <h2 class="govuk-heading-m">@messages("profile.h2")</h2>
+    @govukSummaryList(detailsList)
+
+    <div class="govuk-warning-text">
```

Comment:
I think there is a government component for this - check the other views

Reply:
Updated